packages/@jsii/spec/test/testutil.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs-extra';
+import * as os from 'os';
+import * as path from 'path';
+
+import * as spec from '../lib/assembly';
+
+/**
+ * Construct a temp directory and write an assembly file inside it
+ */
+export class TestAssembly {
```

Comment:
I initially wanted to add `TestJsiiModule` here but that introduces a circular dependency between `@jsii/spec` and `jsii`. Instead, I created something very similar but geared towards my testing needs, what do you think?

In particular, I didn't see the need to do the whole `tmpDir` and `modDir` thing as I don't really care where the assembly file is as long as I can access it.

Reply:
I don't think you need this helper at all.

The test could already be:

* You have an instance of a `spec.Assembly`. You can use `saveAssembly()` to save it to disk, and if you load it back `validateAssembly` doesn't complain.
* You have an instance of a `spec.Assembly`. After you save it, change it and save it again, the fingerprints are different between the two of them.

You could write all of that without this class, and I'm not sure the class makes it more readable.